src/App.tsx

```diff
@@ -1,61 +1,141 @@
+import React, { FormEvent, useState } from 'react';
 import './App.scss';
+import cn from 'classnames';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+
+function getUserById(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const preparedTodo: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
+
+export const getNewId = (array: { id: number }[]) => (
+  Math.max(...array.map(el => el.id)) + 1
+);
+
+export const App: React.FC = () => {
+  const [selectedUserId, setSelectedUserId] = useState(0);
+  const [todos, setTodos] = useState(preparedTodo);
+  const [title, setTitle] = useState('');
+
+  const [selectedUserIdError, setSelectedUserIdError] = useState('');
+  const [titleError, setTitleError] = useState('');
```

Comment:
these values must be boolean, which is probably why the tests fail
Fix this problem